content/benefits/Education-Benefits/release-notes/2023-06-09

```diff
@@ -0,0 +1,20 @@
+Education Benefits API is a new addition to the Lighthouse developer portal.  It replaces the EVSS Education Services API.   
+
+Important: To get production access, you must work for VA or have specific VA agreements. If you have questions, please contact us here; https://developer.va.gov/support/contact-us.
+
+The Education Benefits API allows authorized internal systems to obtain information about a Veteran’s GI Bill (Chapter 33), education benefits eligibility, and usage history.  Learn more about VA education and training benefits.
+
+The API takes in a Veteran’s ICN and returns: 
```

Comment:
Since we have the new suggestion above, I think we can remove some of the repeated information
```suggestion
```